btcwrapper/zmq/client.go

```diff
@@ -22,24 +22,17 @@ var (
 // It handles ZMQ message routing and provides thread-safe access to subscriptions.
 // Must be created with New() and cleaned up with Close().
 type Client struct {
-	// RPC connection
-	rpcClient *rpcclient.Client
-	logger    *zap.SugaredLogger
-
-	// Lifecycle management
-	isClosed int32 // Set atomically
-	wg       sync.WaitGroup
-	quitChan chan struct{}
-
-	// ZMQ configuration
+	rpcClient          *rpcclient.Client
+	logger             *zap.SugaredLogger
+	quitChan           chan struct{}
+	blockEventsChannel chan *btctypes.BlockEvent // Channel for notifying new BTC blocks to relayer
+	zcontext           *zmq4.Context
+	zsubscriber        *zmq4.Socket // Subscriber socket
+	zbackendsocket     *zmq4.Socket // Backend socket for internal communication
 	zeromqEndpoint     string
-	blockEventsChannel chan *btctypes.BlockEvent
-
-	// ZMQ sockets and subscriptions
-	zcontext       *zmq4.Context
-	zsubscriber    *zmq4.Socket  // Subscriber socket
-	subscriptions  Subscriptions // Subscription management
-	zbackendsocket *zmq4.Socket  // Backend socket for internal communication
+	subscriptions      Subscriptions // Subscription management
+	wg                 sync.WaitGroup
+	isClosed           int32 // Set atomically
 }
```

Comment:
let's add `//nolint:fieldalignment`. This struct is only used once, and we don't really save bytes.